packages/kbn-apm-config-loader/src/config.ts

```diff
@@ -84,7 +78,8 @@ export class ApmConfiguration {
         getDefaultConfig(this.isDistributable),
         this.getConfigFromKibanaConfig(),
         this.getDevConfig(),
-        this.getDistConfig()
+        this.getDistConfig(),
+        this.getCIConfig()
```

Comment:
It feels like the ENV shouldn't influence the defaults the way it is now, and instead we should add a new method like `getEnvConfig()` and merge it into the config like the rest of the methods here.

Reply:
The `getCIConfig` aligns with the existing methods in that it augments the defaults based on the environment (development and distribution). Grouping by where the values come from seems doesn't seem like it would make things more clear. Happy to revisit as a whole in a follow-up if you think it makes sense.

Reply:
Sure, it just seems like the `getDefaultConfig()` function was intended to return fixed values, and now it's not really the defaults but the defaults merged with the config from the environment. Not a big deal, obviously.